### PR TITLE
Fix broken i386 simulator tests after the large-types optimization landed

### PR DESCRIPTION
**Pull request by adrian-prantl**
rdar://problem/33761763

### CODE REVIEW

**Comment by adrian-prantl** [comment]
@swift-ci test and merge

File: test/DebugInfo/guard-let.swift [hunk 1]
```diff
@@ -22,6 +22,10 @@ public func f(_ i : Int?)
   use(val)
 }
 
+// With larget type optimizations the string is passed indirectly on i386 so
```
Comment (jrose-apple) [inline test/DebugInfo/guard-let.swift]: Typo: "larget"

Comment (adrian-prantl) [inline test/DebugInfo/guard-let.swift]: thanks

**Comment by adrian-prantl** [comment]
@swift-ci test and merge

**Comment by adrian-prantl** [comment]
@swift-ci please test

**Comment by adrian-prantl** [comment]
@swift-ci please test OS X

**Pull request merged**